pytest.ini

```diff
@@ -1,7 +1,7 @@
 [pytest]
-timeout = 2400
+timeout = 300
```

Comment:
Does this affect all pytests if a timeout is not defined? Wondering if this will break stress tests

Reply:
Yes it should affect all pytests, this PR: https://github.com/tenstorrent/tt-metal/pull/9742 should be setting all the custom timeouts needed